mix/ClientModel.cpp

```diff
@@ -181,7 +188,7 @@ void ClientModel::setupState(QVariantMap _state)
 				transactionSettings.parameterValues.push_back(param);
 			}
 
-			if (transaction.value("executeConstructor").toBool())
+			if (contractId == functionId || functionId == "Constructor")
```

Comment:
Maybe it's possible to continue using transaction.value("executeConstructor").toBool()
rather than : functionId == "Constructor" as this label "Constructor" will certainly be changed.